src/main/menus/bookmarks.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  Menu,
+  nativeImage,
+  NativeImage,
+  MenuItemConstructorOptions,
+  app,
+} from 'electron';
+import { join } from 'path';
+import { IBookmark } from '~/interfaces';
+import { Application } from '../application';
+import { AppWindow } from '../windows/app';
+
+function getPath(file: string) {
```

Comment:
I think it could be removed and the icons could be just added here as constants: https://github.com/wexond/desktop/blob/master/src/renderer/constants/icons.ts

Reply:
I could move that getPath function to constants and use it to export those icons. The challenge with using a regular require like the other icons is that the nativeImage needs an absolute file path and can't use the path served from the webpack-dev-server in dev or a relative path in prod.

Reply:
Ah, I see